app/filters/filters.js

```diff
@@ -1,3 +1,9 @@
+function includeString(text, values) {
+  return values.some(function(val){
+    return text.includes(val);
```

Comment:
I'd suggest using `indexOf` over `includes` because it's ES2016 feature and isn't supported by [some major browsers](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Browser_compatibility).